linux/scripts/dist.sh

```diff
@@ -84,6 +84,9 @@ for proj in ${extra_projects}; do
         kbpvers="keyman-keyboardprocessor-$VERSION"
         cp -a desktop $kbpvers
         cp ../../VERSION.md $kbpvers
+        cp ../../TIER.md $kbpvers
+        mkdir -p $kbpvers/scripts
+        cp ../../resources/shellHelperFunctions.sh $kbpvers/scripts
```

Comment:
Let's add VERSION.md, TIER.md, shellHelperFunctions.sh to .gitignore for this folder.

Reply:
Good idea.